src/main/java/seedu/condonery/logic/commands/AddCommand.java

```diff
@@ -2,59 +2,53 @@
 
 import static java.util.Objects.requireNonNull;
 import static seedu.condonery.logic.parser.CliSyntax.PREFIX_ADDRESS;
-import static seedu.condonery.logic.parser.CliSyntax.PREFIX_EMAIL;
 import static seedu.condonery.logic.parser.CliSyntax.PREFIX_NAME;
-import static seedu.condonery.logic.parser.CliSyntax.PREFIX_PHONE;
 import static seedu.condonery.logic.parser.CliSyntax.PREFIX_TAG;
 
 import seedu.condonery.logic.commands.exceptions.CommandException;
 import seedu.condonery.model.Model;
-import seedu.condonery.model.person.Person;
+import seedu.condonery.model.property.Property;
 
 /**
- * Adds a person to the address book.
+ * Adds a property to Condonery.
  */
 public class AddCommand extends Command {
 
     public static final String COMMAND_WORD = "add";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a property to Condonery. "
             + "Parameters: "
             + PREFIX_NAME + "NAME "
-            + PREFIX_PHONE + "PHONE "
-            + PREFIX_EMAIL + "EMAIL "
             + PREFIX_ADDRESS + "ADDRESS "
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
-            + PREFIX_NAME + "John Doe "
-            + PREFIX_PHONE + "98765432 "
-            + PREFIX_EMAIL + "johnd@example.com "
-            + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
-            + PREFIX_TAG + "friends "
-            + PREFIX_TAG + "owesMoney";
+            + PREFIX_NAME + "PINNACLE@DUXTON "
+            + PREFIX_ADDRESS + "Cantonment Rd, #1G, S085301 "
+            + PREFIX_TAG + "High-End "
+            + PREFIX_TAG + "Available";
 
-    public static final String MESSAGE_SUCCESS = "New person added: %1$s";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book";
+    public static final String MESSAGE_SUCCESS = "New property added: %1$s";
+    public static final String MESSAGE_DUPLICATE_property = "This property already exists in Condonery";
 
-    private final Person toAdd;
+    private final Property toAdd;
 
     /**
-     * Creates an AddCommand to add the specified {@code Person}
+     * Creates an AddCommand to add the specified {@code Property}
      */
-    public AddCommand(Person person) {
-        requireNonNull(person);
-        toAdd = person;
+    public AddCommand(Property property) {
+        requireNonNull(property);
+        toAdd = property;
     }
 
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
 
-        if (model.hasPerson(toAdd)) {
-            throw new CommandException(MESSAGE_DUPLICATE_PERSON);
+        if (model.hasProperty(toAdd)) {
+            throw new CommandException(MESSAGE_DUPLICATE_property);
```

Comment:
might want to capitalize this to keep it consistent!

Reply:
Done, thanks!